src/js/app/App.js

```diff
@@ -0,0 +1,48 @@
+import routes from 'js/utils/routes';
+import { Header } from 'js/components';
+import { Home, Lines, Login, Sections, NotFound, Stations } from 'js/pages';
+import render from 'js/utils/render';
+import ajax from 'js/api';
+
+const { HOME, LINES, LOGIN, SECTIONS, STATIONS } = routes;
+
+const App = (async () => {
+  let stations = await ajax.getStations();
+  const setStations = newStations => {
+    stations = newStations;
+    render();
+  };
+  return () => {
+    const $app = document.createElement('div');
+    $app.className = 'd-flex justify-center mt-5 w-100';
+
+    const $appInnerContainner = document.createElement('div');
+    $appInnerContainner.className = 'w-100';
+
+    const $mainContainer = document.createElement('main');
+    $mainContainer.className = 'mt-10 d-flex justify-center';
+
+    const routesMatchComponents = [
+      { path: HOME, component: Home },
+      { path: LINES, component: Lines },
+      { path: LOGIN, component: Login },
+      { path: SECTIONS, component: Sections },
+      { path: STATIONS, component: Stations },
+    ];
+
+    const pathMatchRoute = routesMatchComponents.find(({ path }) => path === window.location.pathname);
+
```

Comment:
배열에 저장하는 것보다 아예 객체에 저장하시는게 어떠신가요 ?? 그러면 더 빠르게 찾을 수 있을 것 같은데요!ㅎㅎ 어차피 라우팅 컴포넌트가 코드 내부에서 추가되거나 하지는 않으니까요~~
```suggestion
    const routesMatchComponents = {
       HOME : Home,
       LINES : Lines,
       LOGIN: Login,
       SECTIONS: Sections,
       STATIONS: Stations
    }
    const pathMatchRoute = routesMatchComponents[path]

```

Reply:
그렇네요! path에 따라 컴포넌트를 참조하는게 아주 간결해졌네요ㅎㅎ 감사합니다